trifid/Chart.yaml

```diff
@@ -1,6 +1,7 @@
-apiVersion: v1
-appVersion: "2.3.2"
-description: Trifid Helm chart
-home: "https://github.com/zazuko/trifid"
+apiVersion: v2
```

Comment:
`v2` is Helm 3 only apparently: https://helm.sh/docs/topics/charts/#the-apiversion-field